test/extended/util/cli.go

```diff
@@ -66,6 +82,13 @@ func (c *CLI) Username() string {
 	return c.username
 }
 
+// AsAdmin changes current config file path to the admin config.
+func (c *CLI) AsAdmin() *CLI {
```

Comment:
Bikeshedding alarm on? Come on guys, let's focus on what is needed to merge this... I don't want to see @mnagy working on this until Christmas :santa: